resources/web/style/settings_modal.pcss

```diff
@@ -0,0 +1,51 @@
+#settings_modal {
+  display: inline-block;
+  position: relative;
+  margin: 15% auto;
+  width: 600px;
+  background: #f8f8f8;
+  border: 1px solid #ddd;
+  border-radius: 10px;
+  padding: 2em 4em 1em;
+  text-align: left;
+  box-shadow: 5px 7px 7px #ccc;
+  &_bg {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: rgba(255,255,255,0.8);
+    z-index: 99999;
+    text-align: center;
+  }
+  label {
+    display: block;
+    margin: 0.5em 0;
+    color: #2b4590;
+  }
+  input {
+    width: 100%;
+    display: block;
+    margin-bottom: 0.1em;
+    font-family: monospace;
+  }
+  button {
+    margin: 0.5em 0.5em 1em 0;
+    background-color: #00a9e5;
+    color: white;
+    border: none;
+  }
+}
+
+.settings_modal-close {
+  position: absolute;
+  top: 2px;
+  right: 10px;
+  font-size: 20px;
+  color: #ccc;
+  &:hover {
+    color: black;
```

Comment:
This used to be `opacity: black` but it wasn't doing anything. I think we just intended to turn the x black though.